active.go

```diff
@@ -32,7 +33,7 @@ type ActiveOption struct {
 // FindByOwner fetches active builds based on the owner's name
 //
 // Travis CI API docs: https://developer.travis-ci.com/resource/active#for_owner
-func (as *ActiveService) FindByOwner(ctx context.Context, owner string, opt *ActiveOption) ([]*Build, *http.Response, error) {
```

Comment:
Hmm, I personally think this is kind of confusing that some methods return `*http.Response` and the other return `*Response`.